libraries/botframework-connector/package.json

```diff
@@ -50,7 +50,8 @@
   },
   "scripts": {
     "build": "tsc",
-    "postbuild": "shx cp lib/index.js lib/browser.js | browserify -s BFC --debug lib/browser.js | exorcist lib/browser.js.map | sponge lib/browser.js",
+    "browserify": "browserify -s BFC --debug lib/browser.js | exorcist lib/browser.js.map | sponge lib/browser.js",
+    "postbuild": "rimraf lib/browser.* && shx cp lib/index.js lib/browser.js && npm run browserify",
```

Comment:
Test flakes were caused by `lib/browser.js` and/or `lib/browser.js.map` files from previous builds. Cleaning them up before re-browserifying fixes that.